python/fusion_engine_client/messages/defs.py

```diff
@@ -111,26 +111,40 @@ class MessageType(IntEnum):
 
     RESERVED = 20000
 
+    @classmethod
+    def get_type_string(cls, type, include_value=True, raise_on_unrecognized=False):
+        return MessageType.static_to_string(type, include_value=include_value,
+                                            raise_on_unrecognized=raise_on_unrecognized)
 
     @classmethod
-    def get_type_string(cls, type):
+    def static_to_string(cls, value, include_value=True, raise_on_unrecognized=False):
         try:
-            if isinstance(type, str):
-                # Convert a string name to a message type (e.g., 'POSE' -> MessageType.POSE).
-                type = MessageType[type.upper()]
-            else:
-                # Convert an int to a MessageType. If `type` is already a MessageType, it'll pass through.
-                type = MessageType(type)
-
-            return '%s (%d)' % (type.name, type.value)
-        except (KeyError, ValueError):
+            return super().static_to_string(value, include_value=include_value, raise_on_unrecognized=True)
```

Comment:
I'm a bit confused what the purpose of this call is. The parent of this class is IntEnum which doesn't define this?